### ISSUE TITLE
the result in the arxiv v1 version is inconsistent with ICCV19 version

### ISSUE THREAD

**Issue by FingerRec** [mit-han-lab/temporal-shift-module#38]
Thanks for your good work! 

I follow TSM early after submit to arxiv first time. And I find the result in older version https://arxiv.org/pdf/1811.08383v1.pdf, Table 2,
`TSM ResNet50  16 65G 24.3M 44.8 74.5`
 while in your ICCV paper under the same setting, the result is . 
`TSM ResNet50  16 65G 24.3M 47.2 77.1`

 While there is no difference in performance between kinetics pretrained model in these two versions, If the reason is using different hyperparmater or training the data more sufficient?
Looking forward to your reply! :)

**Comment by tonylins** [comment]
Hi, the main difference that leads to an accuracy increase on Something-Something is that we set a proper learning rate in the new version (the most important factor is to enlarge the initial learning rate). We also switch to ImageNet pre-training, since we find that Kinetics pre-training does not help better. While for Kinetics, we used a proper learning rate in the first version.
